Mirror/Runtime/ExponentialMovingAverage.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+namespace Mirror
+{
+    // implementation of N-day EMA
+    // it calculates an exponential moving average roughy equivalent to the last n observations
+    public class ExponentialMovingAverage
```

Comment:
Maybe add 'https://en.wikipedia.org/wiki/Moving_average' as comment above class?
Many might not know that this concept had a name